src/components/calculator/Calculator.test.js

```diff
@@ -0,0 +1,18 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Calculator from './Calculator';
+
+describe('Calculator page testing', () => {
+  it('snapshot matching', () => {
+    const tree = renderer.create(<BrowserRouter><Calculator /></BrowserRouter>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('Document testing', () => {
```

Comment:
- [ ] Please as the last reviewer said include tests for user interactions (e.g click, etc) using the React Testing Library, I see that you are not using the button tags but you are using a div tag as a button which is not semantic but in this project, you should do some tests on the user interaction with the buttons.
![image](https://user-images.githubusercontent.com/74744787/222810907-689df2a6-0a29-42d9-acd6-c49e5f29c872.png)